Python/pylifecycle.c

```diff
@@ -345,20 +345,6 @@ initimport(PyInterpreterState *interp, PyObject *sysmod)
     return _Py_INIT_OK();
 }
 
-static _PyInitError
```

Comment:
This needs to stay - the bug is that `initimport` is trying to install the external importers that rely on `sys.path` (et al) already being configured properly.